lua/tests/gmod/classes/file/Tell.lua

```diff
@@ -0,0 +1,24 @@
+return {
+    groupName = "File:Tell",
+
+    cases = {
+        {
+            name = "Exists on the File meta table",
+            func = function()
+                local meta = FindMetaTable( "File" )
+                expect( meta.Tell ).to.beA( "function" )
+            end
+        },
+
+        {
+            name = "Returns the right value",
+            func = function()
+                local a = GetTestFile( "Write" )
+
+                a:Skip( 12 )
+
+                expect( a:Tell() ).to.equal( 12 )
```

Comment:
nit: When two steps of the test use the same value, I like to assign it to a variable to show that they are indeed using the _same value_, and that it wasn't just coincidence they were the same

```suggestion
                local a = GetTestFile( "Write" )
                local skipAmount = 12

                a:Skip( skipAmount )

                expect( a:Tell() ).to.equal( skipAmount )
```